openapi/schemas/user/ImportTimeRegistrationRequest.yaml

```diff
@@ -17,3 +17,5 @@ items:
       type: array
       items:
         $ref: TagTimeRegistrationRequest.yaml
+    description:
```

Comment:
`taskId` should no longer be mandatory in this. Specify that if `taskId =null` then `description` must be included.

Missing same changes in `TimeRegistrationRequest.yaml`